.orchestra/config/components/clang_release.yml

```diff
@@ -48,7 +48,7 @@ install: |
 build_dependencies:
   - cmake
 dependencies:
-  - gcc-host-toolchain
+  - toolchain/host/cxx-toolchain
```

Comment:
I think this is wrong. `toolchain/host/cxx-toolchain` depends on `clang-release`...
Please do not submit non-draft PRs that are untested.